tests/simple_child.rs

```diff
@@ -45,7 +45,7 @@ fn simple_child() {
     let node1 = taffy
         .new_with_children(
             taffy::style::FlexboxLayout {
-                size: taffy::geometry::Size { width: Dimension::Undefined, height: Dimension::Undefined },
+                size: taffy::geometry::Size { width: Dimension::Auto, height: Dimension::Auto },
```

Comment:
This should be `None` instead of `Auto.` Which means I need to change `Size` to be `Option<Dimension>` here as well